script.js

```diff
@@ -51,29 +53,24 @@ buttonElement.addEventListener('click', function (event) {
 });
 
 buttonRight.addEventListener('click', function (){
-    move('right');
+    moveCarouselSlides('right');
 }); 
 buttonLeft.addEventListener('click', function (){
```

Comment:
Why do you need a function that calls another function here? You can just directly call moveCarouselSlides

Reply:
Yes it's true. But it was easier for me to split the task into several steps-functions.

Reply:
Ok, deleted this extra function call.